python/GafferSceneUI/StandardOptionsUI.py

```diff
@@ -77,16 +77,20 @@ def __cameraSummary( plug ) :
 
 	return ", ".join( info )
 
-def __purposeSummary( plug ) :
+def __renderSetSummary( plug ) :
 
+	info = []
 	if plug["includedPurposes"]["enabled"].getValue() :
 		purposes = plug["includedPurposes"]["value"].getValue()
-		if purposes :
-			return ", ".join( [ p.capitalize() for p in purposes ] )
-		else :
-			return "None"
+		info.append( "Purposes {}".format( ", ".join( [ p.capitalize() for p in purposes ] ) if purposes else "None" ) )
+	if plug["inclusions"]["enabled"].getValue() :
+		info.append( "Inclusions {}".format( plug["inclusions"]["value"].getValue() ) )
+	if plug["exclusions"]["enabled"].getValue() :
+		info.append( "Exclusions {}".format( plug["exclusions"]["value"].getValue() ) )
+	if plug["additionalLights"]["enabled"].getValue() :
+		info.append( "Lights {}".format( plug["additionalLights"]["value"].getValue() ) )
 
-	return ""
+	return ", ".join( info )
```

Comment:
Nitpick : Joining purposes with a comma and then joining that with other things with a comma is confusing :

![image](https://github.com/GafferHQ/gaffer/assets/1133871/8afd78ff-f2f5-4b1c-b99f-c363590c45b6)

Should we join purposes with a `/` instead? Or `&`?